### PR TITLE
i18n(ko-KR): translate `customization.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

translate `customization.mdx`

### CODE REVIEW

File: docs/src/content/docs/ko/guides/customization.mdx [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+---
+title: '404'
+template: splash
+editUrl: false
+hero:
+  title: '404'
+  tagline: 페이지를 찾을 수 없습니다. URL을 확인하거나 검색창을 사용해 보세요.
+---
+```
+````
```
Comment (delucis) [inline docs/src/content/docs/ko/guides/customization.mdx:256]: Looks like we got an extra `md` code block wrapper here:

| English | Korean |
|---|---|
|  image |  image |

````suggestion
```md
---
title: '404'
template: splash
editUrl: false
hero:
  title: '404'
  tagline: 페이지를 찾을 수 없습니다. URL을 확인하거나 검색창을 사용해 보세요.
---
```
````

File: docs/src/content/docs/ko/guides/customization.mdx [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+    		}),
+    	],
+    });
+    ```
+
+    Fontsource는 각 글꼴에 대해 여러 CSS 파일을 제공합니다. 다른 굵기 및 스타일을 포함하는 방법이 궁금하다면 [Fontsource 문서](https://fontsource.org/docs/getting-started/install#4-weights-and-styles)를 참조하세요.
+
+### 글꼴 사용
+
+설정한 글꼴을 사이트에 적용하려면 [사용자 정의 CSS 파일](/ko/guides/css-and-tailwind/#custom-css-styles)에서 선택한 글꼴 이름을 사용하세요. 예를 들어 Starlight의 기본 글꼴을 변경하려면 사용자 정의 속성인 `--sl-font`를 설정하세요.
```
Comment (delucis) [inline docs/src/content/docs/ko/guides/customization.mdx:379]: Like the link on line 400, this one should use the `#` fragment that matches the translation:

```suggestion
설정한 글꼴을 사이트에 적용하려면 [사용자 정의 CSS 파일](/ko/guides/css-and-tailwind/#사용자-정의-css-스타일)에서 선택한 글꼴 이름을 사용하세요. 예를 들어 Starlight의 기본 글꼴을 변경하려면 사용자 정의 속성인 `--sl-font`를 설정하세요.
```

**Review by delucis — COMMENTED** [review]
Thank you @jsparkdev — this looks great! Noticed two small details, but otherwise ready to go 🚀

File: docs/src/content/docs/ko/guides/customization.mdx (hunk 2 above)
Comment (jsparkdev) [reply docs/src/content/docs/ko/guides/customization.mdx]: thank you

File: docs/src/content/docs/ko/guides/customization.mdx (hunk 1 above)
Comment (jsparkdev) [reply docs/src/content/docs/ko/guides/customization.mdx]: Thanks for the good point! 🙂

**Review by delucis — APPROVED** [review]
Thanks for making those fixes @jsparkdev! 🎊

**Pull request merged**